helm/templates/entity-service-config.yaml

```diff
@@ -7,20 +7,27 @@ metadata:
 data:
   application.conf: |-
     entity.service.config = {
-      entity-service {
-        {{- $dst := .Values.entityServiceConfig.dataStoreType }}
+      document.store {
+        {{- if .Values.database }}
+        {{- $dst := .Values.database.type }}
         dataStoreType = {{ $dst }}
+        appName = {{ .Values.service.name }}
         {{ $dst }} {
-          {{- range $key, $value := (index .Values "entityServiceConfig" (printf "%s" $dst)) }}
+          {{- range $key, $value := (index .Values "database" (printf "%s" $dst)) }}
           {{- if $value }}
-            {{- if eq $key "password" }}
+            {{- if hasPrefix "${?" (printf "%s" $value) }}
           {{ $key }} = {{ $value }}
             {{- else }}
+              {{- if eq $key "endpoints" }}
```

Comment:
do we need to special case this? Can we we toJson everything (I'm assuming a standard string would just quote but I'm not sure)

Reply:
Did some experiments earlier, but couldn't get it working. Also, repeated after this comment, but unable to make it working without special casing. @nikhil-rajesh Suggest if there are any other easy/precise ways.